### DISCUSSION TITLE
每次打开的应用界面展示大小都太小了

### DISCUSSION THREAD

**Discussion by lpxks** [clash-verge-rev/clash-verge-rev#2195] [Q&A]
每次都要拉伸 能不能固定软件窗口大小有没有什么设置可以固定大小 不要每次手动拉升。 我记得前几个版本都没有这个问题的 是bug吗? 下面图片还是放大了的效果。我使用软件的版本是2.0.1 系统windows10
![image](https://github.com/user-attachments/assets/1eb96006-866b-4929-95fa-7412d1b6fd8a)

**Comment by MuFengss** [comment]
对啊，我也遇到这个问题，调整好的大小，开关软件每次都重置了。
